src/dmd/backend/cgen.d

```diff
@@ -131,88 +116,84 @@ code *cat(code *c1,code *c2)
 
 code *gen(code *c,code *cs)
 {
-#ifdef DEBUG                            /* this is a high usage routine */
```

Comment:
Retain the comment.

Reply:
The comment is obsolete. The CodeBuilder.gen() retains the comment. The older gen() is rarely called.